ddl/ddl.go

```diff
@@ -1630,11 +1633,12 @@ func processAllJobs(process func(*sess.Session, *model.Job, model.AdminCommandOp
 			err = process(ns, job, byWho)
 			if err != nil {
 				jobErrs[job.ID] = err
-				ns.Rollback()
-				return jobErrs, err
+				continue
 			}
+
 			err = updateDDLJob2Table(ns, job, true)
 			if err != nil {
+				// unexpected error on updating the job table, should roll it back
```

Comment:
Ditto.

Reply:
It looks like better to handle two different kind of failure.